system/lib/libc/musl/src/temp/__randname.c

```diff
@@ -8,9 +8,10 @@ char *__randname(char *template)
 	int i;
 	struct timespec ts;
 	unsigned long r;
+	static unsigned int seed = 0;
 
 	__clock_gettime(CLOCK_REALTIME, &ts);
-	r = ts.tv_nsec*65537 ^ (uintptr_t)&ts / 16 + (uintptr_t)template;
+	r = ts.tv_nsec*65537 ^ (uintptr_t)&ts / 16 + (uintptr_t)template + seed++;
```

Comment:
Can you mark with `/* XXX EMSCRIPTEN ...  */` and maybe try to keep the changes isolated from the original code.  How about:

```
/* XXX EMSCRIPTEN: avoid repeating the same result when __clock_gettime does not change between calls. */
static unsigned int seed = 0;
r += seed++;
```

Reply:
understood, updating it